File: ScannerVB/Model/FicSource.vb
Public Class FicSource

    Property NomIntFicSource As String
    Property NomExtFicSource As String
    Property Emplacement As String
    Property TypeFicSource As TypeSource.TypeFichier

    Property NombreDeCreateObject As Integer = 0
    Property NombreDApi32 As Integer = 0
    Property NombreDeExcelApp As Integer = 0
    Property NombreDeWordApp As Integer = 0
    Property NombreDeOfficeouAutre As Integer = 0
    Property NombreDeUserControles As Integer = 0
    Property NombreDeLignesActives As Integer = 0

    Property ListofRegeX As New List(Of RegexFound)

    Property NomApplication As String
    Property NomDuFichierVBP As String

End Class

File: ScannerVB6/ScannerVB6/Model/FicSource.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScannerVB6.Model
{
    public class FicSource
    {
        public string? NomIntFicSource { get; set; }
        public string? NomExtFicSource { get; set; }
        public string? Emplacement { get; set; }
        public string? NomApplication { get; set; }
        public string? NomDuFichierVBP { get; set; }

        public TypeSource.TypeFichier TypeFicSource { get; set; }

        public int NombreDeCreateObject { get; set; } = 0;
        public int NombreDApi32 { get; set; } = 0;
        public int NombreDeExcelApp { get; set; } = 0;
        public int NombreDeWordApp { get; set; } = 0;
        public int NombreDeOfficeouAutre { get; set; } = 0;
        public int NombreDeUserControles { get; set; } = 0;
        public int NombreDeLignesActives { get; set; } = 0;

        public List<RegexFound> ListofRegeX { get; set; } = new List<RegexFound>();



    }
}
